mod/bookmarks/elgg-plugin.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+	'actions' => [
+		'logged_in' => [
+			'bookmarks/save' => 'bookmarks/save.php',
+			'bookmarks/delete' => 'bookmarks/delete.php',
+		],
+	],
+	'views' => [
+		'default' => [
+			'bookmarks/' => __DIR__ . '/graphics',
```

Comment:
@hypeJunction Editing these files with a CLI tool might be tricky due to constants like this. Just thinking ahead. Fine for now.